people.html

```diff
@@ -119,45 +126,24 @@ <h1>Manideepa Saginatham</h1>
 
     <section class="py-3">
       <div class="container">
-        <h1>Susmita Padala</h1>
```

Comment:
These former students need to still be kept within the website, but probably in a **Former Students** section. This can be fixed in the next round of feature updates though.